docs/sources/release-notes/v2-4.md

```diff
@@ -47,16 +49,38 @@ Here is a list of limit defaults that have changed:
 
 We have enabled the [in-memory FIFO cache by default](https://github.com/grafana/loki/pull/4519) when no existing external cache is configured. This helps improve Loki's performance, however it will also allow Loki to consume more memory.
 
+### v2.4.2 changes to defaults
+
+Here is a list of limit defaults that have changed in v2.4.2:
+
+| config | new default | old default |
+| --- | --- | --- |
+| parallelise_shardable_queries | true | false |
+| split_queries_by_interval | 30m | 0s |
+| query_ingesters_within | 3h | 0s |
+| max_chunk_age | 2h | 1h |
+| max_concurrent | 10 | 20 |
+
 ## Bug fixes
 
 ### 2.4.2 bug fixes
-2.4.2 fixes these important bugs:
-
-* ADD BUG FIX HERE
+2.4.2 fixes these bugs:
+
+- [PR 4968](https://github.com/grafana/loki/pull/4968) **trevorwhitney**: Fixes a bug that caused a communication error between components during a query.
```

Comment:
```suggestion
- [PR 4968](https://github.com/grafana/loki/pull/4968) **trevorwhitney**: Fixes a bug where a ruler could be wrongly returned when querying ingesters, causing an internal server error with `code = Unimplemented`. 
```

Reply:
Revised the item for PR 4968 a bit different than given in the suggestion. @DylanGuedes , please re-review.

Reply:
looks great!